tensorpack/dataflow/imgaug/crop.py

```diff
@@ -127,8 +137,10 @@ def _get_augment_params(self, img):
     def _augment(self, img, newbox):
         return newbox.roi(img)
 
-    def _augment_coords(self, coords, param):
-        raise NotImplementedError()
+    def _augment_coords(self, coords, newbox):
```

Comment:
Some sort of random bounding box perturbation is definitely helpful, but the problem is there isn't a standard way of doing this. If something isn't a standard, I think it's best to let users write their own version of it than to provide some arbitrary implementation. I might just mark this augmentor as deprecated.